### PR TITLE
Refs BOT-579 ~ Line sections implemented in ptObjectFilter.

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

This PR implement filter of disruption by POST request

## How to test

New parameter `line_sections` should work in `ptObjectFilter`
More information in the new swagger: [Chaos Swagger](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/feature_BOT-579_line_sections/documentation/swagger.yml#/Search/post_disruptions__search)

### CODE REVIEW

File: chaos/models.py [hunk 1]
```diff
@@ -568,21 +567,28 @@ def all_with_post_filter(
...
-                uris = uris + objectId
+            for key, objectIds in ptObjectFilter.iteritems():
+                object_type = key[:-1]
+                object_types.append(object_type)
+                uris = uris + objectIds
+                if object_type == 'line_section':
+                    line_section_uris = [objectId + ':%' for objectId in objectIds]
+
             query = cls.query.filter(
                 and_(
```
Comment (pthegner) [inline chaos/models.py]: or_(and_(                     cls.impacts.any(Impact.objects.any(PTobject.type.in_(object_types))),
                    cls.impacts.any(Impact.objects.any(PTobject.uri.in_(uris))),
                    cls.impacts.any(Impact.objects.any(
                        or_(*[PTobject.uri.like(objectId) for objectId in line_section_uris])
                    )))

File: chaos/models.py [hunk 2]
```diff
@@ -568,32 +567,38 @@ def all_with_post_filter(
...
+                    or_(*[PTobject.uri.like(objectId) for objectId in line_section_uris])
+                )))
             )
-
         return cls.get_query_with_args(
             contributor_id=contributor_id,
             publication_status=publication_status,
             ends_after_date=ends_after_date,
             ends_before_date=ends_before_date,
+            line_section=True,
```
[the remark this answers is not in the document]
Comment (pthegner) [reply chaos/models.py]: You should send line_section like before here.

File: chaos/models.py [hunk 3]
```diff
@@ -567,33 +566,40 @@ def all_with_post_filter(
...
 
         if uri is None and ptObjectFilter is not None:
-            for key, objectId in ptObjectFilter.iteritems():
-                object_types.append(key[:-1])
-                uris = uris + objectId
+            for key, objectIds in ptObjectFilter.iteritems():
+                object_type = key[:-1]
+                object_types.append(object_type)
+                uris = uris + objectIds
+                if object_type == 'line_section':
```
Comment (pthegner) [inline chaos/models.py]: I'm really unsure about that

**Comment by Netmisa** [comment]
Jenkins could yoou test this please

**Comment by pthegner** [comment]
retest this please

**Comment by Netmisa** [comment]
Jenkins could you test this please

**Pull request merged**